drivers/windows/overlay/joinleave_windows.go

```diff
@@ -93,6 +93,10 @@ func (d *driver) EventNotify(etype driverapi.EventType, nid, tableName, key stri
 	d.peerAdd(nid, eid, addr.IP, addr.Mask, mac, vtep, true)
 }
 
+func (d *driver) DecodeTableEntry(tablename string, key string, value []byte) (string, map[string]string) {
+	return "", nil
+}
```

Comment:
Windows overlay driver also uses the networkdb. I guess it is valid to have this function populated ?